mesonbuild/ast/interpreter.py

```diff
@@ -243,10 +244,10 @@ def __init__(self, source_root: str, subdir: str, subproject: SubProject, subpro
                            'debug': self.func_do_nothing,
                            })
 
-    def _unholder_args(self, args: _T, kwargs: _V) -> T.Tuple[_T, _V]:
+    def _unholder_args(self, args: T.Any, kwargs: T.Any) -> T.Tuple[T.Any, T.Any]:
```

Comment:
I doubt you want to do this.   @dcbaker probably has a better suggestion for you.

Reply:
I don't really see an alternative :(

Reply:
I could try, but it's not easy and I think it's something we should do after merging this PR if at all. This PR is already big enough. I could add a # todo comment if you want to.

Keep in mind that before this PR, this file was not typechecked at all.